app/models/concerns/catalog/clone.rb

```diff
@@ -2,12 +2,12 @@ class Catalog < ApplicationRecord
   module Clone
     extend ActiveSupport::Concern
 
-    def clone!
+    def clone!(new_slug)
```

Comment:
je trouve plus clean d'utiliser les attribut nommé (c'est peut etre pas le bon terme) :
```suggestion
    def clone!(slug: new_slug)
```

Reply:
yes